.env.test

```diff
@@ -1,6 +1,7 @@
 # Client settings
 NEXT_PUBLIC_APP_HOST=localhost:3000
 NEXT_PUBLIC_APP_USE_TLS=0
+NEXT_PUBLIC_CYPRESS=true
```

Comment:
I would rename this to `CYPRESS` and set it's value to `1` for consistency (compare with `NEXT_PUBLIC_APP_USE_TLS`.

The `NEXT_PUBLIC_` prefix means that the variable will be compiled into the front-end bundle, i.e. be exposed to the browser. Although there is no security issue in exposing this particular variable to the browser, I think it only needs to be available to server-side code and we should avoid creating a pattern of exposing more than necessary.